internal/debug/flags.go

```diff
@@ -55,6 +55,7 @@ var (
 	memprofilerateFlag = cli.IntFlag{
 		Name:  "memprofilerate",
 		Usage: "Turn on memory profiling with the given rate",
+		Value: runtime.MemProfileRate,
```

Comment:
This doesn't automatically turn on profiling, right? Also shouldn't something similar be done for `blockprofilerate`?